webui/src/pages/repositories/wizard/wizard.jsx

```diff
@@ -0,0 +1,101 @@
+import React, {useState} from "react";
+import StepWizard from "react-step-wizard";
+import {ProgressBar} from "react-bootstrap";
+import Container from "react-bootstrap/Container";
+import Row from "react-bootstrap/Row";
+import Col from "react-bootstrap/Col";
+
+const transitionDefaults = {
+    enterRight: "",
+    enterLeft: "",
+    exitRight: "",
+    exitLeft: "",
+    intro: "",
+}
+
+const Wizard = ({
+                    hashEnabled=false,
+                    transitions=transitionDefaults,
+                    progressBar = false,
+                    skipButton = false,
```

Comment:
```suggestion
                    isShowProgressBar = false,
                    isShowSkipButton = false,
```

Reply:
```suggestion
                    showProgressBar = false,
                    showSkipButton = false,
```
?

Reply:
Was going for the "is" prefix with booleans, but I can live with that.